### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by AaronVL**
@BryanCable Please review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,6 +10,35 @@
...
+	      </button> <!-- Hamburger Menu Ends  -->
+	      <a class="navbar-brand" href="#">
+	      	<img src="img/name-01.png" alt="Restaurante" class="">
+	      </a>
+	    </div>
+
+	    <!-- Collect the nav links, forms, and other content for toggling -->
+	    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+	      <ul class="nav navbar-nav navbar-right">
+					<li class="active"><a href="#">Home <span class="sr-only">(current)</span></a></li>
```
Comment (BryanCable) [inline index.html:32]: Line is spaced to far, please delete the extra space.

File: index.html [hunk 2]
```diff
@@ -10,6 +10,35 @@
...
+	        <li><a href="#">Our Story</a></li>
+	        <li><a href="#">Review</a></li>
+					<li><a href="#">Menu</a></li>
```
Comment (BryanCable) [inline index.html:35]: Please remove extra white space.

File: index.html [hunk 3]
```diff
@@ -10,6 +10,35 @@
...
+					<li><a href="#">Contact Us</a></li>
```
Comment (BryanCable) [inline index.html:36]: Please remove extra space

**Review by BryanCable — CHANGES_REQUESTED** [review]
Remove extra spaces.

File: index.html (hunk 2 above)
Comment (AaronVL) [reply index.html:35]: looks good on atom

**Comment by BryanCable** [comment]
ok to merge

**Pull request merged**